pkg/operator/ceph/controller/spec.go

```diff
@@ -813,6 +813,30 @@ func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Contai
 	}
 }
 
+// rgw operations will be logged in sidecar rgw-ops-log
+func RgwOpsLogContainer(daemonID, ns string, c cephv1.ClusterSpec, rgwOpsLogResources v1.ResourceRequirements) *v1.Container {
+
+	// Generate ops log file name based on daemon ID
+	opsLogFile := fmt.Sprintf("ops-log-%s.log", daemonID)
+
+	return &v1.Container{
+		Name: "rgw-ops-log",
+		Command: []string{
+			"/bin/bash",
+			"-m", // Terminal job control, allows job to be terminated by SIGTERM
+			"-c", // Command to run
+			fmt.Sprintf("tail -n+1 -F %s", path.Join(config.VarLogCephDir, opsLogFile)),
```

Comment:
Do we need to handle log file rotation too?

Reply:
One of the Ceph docs notes that the ops log can be extremely large. I would therefore suggest that we plan how these logs can be managed by the log rotator. Otherwise, I believe it won't be long before users complain of host disk usage being overconsumed.

@parth-gr has done a fair amount of work on log rotation and might have thoughts/suggestions.

Reply:
@parth-gr does all the files in rgw container have log rotation or does it need to updated for the ops log file as well, do you have any insights here?

Reply:
@ideepika yes the logrotater is configured for rgw pod https://github.com/rook/rook/blob/master/pkg/operator/ceph/object/spec.go#L158